Readme.md

```diff
@@ -112,11 +112,11 @@ pod 'Moya'
 
 # or 
 
-pod 'Moya/RxSwift'
+pod 'Moya/RxSwift', '~> 10.0'
```

Comment:
Any reason why we didn't do the core Moya pod as well?

Reply:
Hey, 

sorry, it is my first ever contribution and I may have misunderstood the issue. Is there another doc where Cocoapods Installation is mentioned and would need to be updated? From what I understood, it was simply to mention the version to use in the Cocoapods Installation section.